<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SurfaceExtractView</class>
 <widget class="QWidget" name="SurfaceExtractView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>427</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QmitkDataStorageComboBox" name="ImageSelector"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Smooth Times</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ImageSmoothTimes">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>40</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ImageLargestConnected">
        <property name="text">
         <string>Largest Connected</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ImageExtractBtn">
        <property name="text">
         <string>Extract</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Surface</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QmitkDataStorageComboBox" name="SurfaceSelector"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Smooth Times</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="SurfaceSmoothTimes">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>40</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SurfaceSmoothBtn">
        <property name="text">
         <string>Smooth</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Surface</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QmitkDataStorageComboBox" name="SimplifySurfaceSelector"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Rate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="SimplifyRate">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Mode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="SimplifyMode">
        <item>
         <property name="text">
          <string>Standard</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Quadric</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SimplifyBtn">
        <property name="text">
         <string>Simplify</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBox</class>
   <extends>QComboBox</extends>
   <header>QmitkDataStorageComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
